### ISSUE TITLE
AlexNet experiments report

### ISSUE THREAD

**Issue by mrgloom** [NVIDIA/DIGITS#1089]
I want to post some results I have got using variations of AlexNet architecture and [this dataset](https://www.kaggle.com/c/dogs-vs-cats) 20k for training and 5k for testing.

Original run: I get about 93.65% in 30 epochs, as we can see on learning curve network start overfitting.
![original](https://cloud.githubusercontent.com/assets/4003908/18649076/98513cc2-7ece-11e6-8d0b-e7ce79375da1.png)

Just to reproduce original run results as sanity check of the repeatability of the result(I have recreate db from same data): I get about 94.00% in 30 epochs.
![reproduce](https://cloud.githubusercontent.com/assets/4003908/18649214/548f1f80-7ecf-11e6-8034-60da38d81714.png)

Remove mean image subtraction: I get about  88.37% in 30 epochs, but maybe if it train more it will get better accuracy? but anyway convergence is slower.
![no-mean-image](https://cloud.githubusercontent.com/assets/4003908/18649275/af1efee8-7ecf-11e6-869a-2bf5fbf05bcb.png)

Use pixel subtraction:  I get about 93.80% in 30 epochs, result is about the same as using mean image subtraction.
![pixel-sub](https://cloud.githubusercontent.com/assets/4003908/18649356/2cfae49e-7ed0-11e6-9d68-d2fb84b7b4f7.png)

Remove LRN(Local Response Normalization) layers: I get about 93.76% in 30 epochs, seems it's safe to remove LRN layers.
![remove-lrn](https://cloud.githubusercontent.com/assets/4003908/18650238/05f7fbb6-7ed5-11e6-8296-db5d83afb60c.png)

**Comment by lukeyeager** [comment]
Are you reporting a bug? If not, this type of post is a better fit for the user group.
